data/transactions/logic/eval_test.go

```diff
@@ -536,34 +537,46 @@ int 1                   // ret 1
 `, 2)
 }
 
+func TestUint128(t *testing.T) {
+	x := uint128(0, 3)
+	require.Equal(t, x.String(), "3")
+	x = uint128(1, 3)
+	require.Equal(t, x.String(), "18446744073709551619")
+	x = uint128(1, 5)
+	require.Equal(t, x.String(), "18446744073709551621")
```

Comment:
add uint128(0, 0) and uint128(uint64(-1), uint64(-1)) tests?

Reply:
done. I assume you mean a test for largest uint64, I used
x = uint128(^uint64(0), ^uint64(0)) // maximum uint128 = 2^64-1
require.Equal(t, x.String(), "340282366920938463463374607431768211455")